auto-value-gson-runtime/src/main/java/com/ryanharter/auto/value/gson/SerializableJsonElement.java

```diff
@@ -0,0 +1,47 @@
+// modified by mapbox
+package com.ryanharter.auto.value.gson;
+
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.Objects;
+
+public class SerializableJsonElement implements Serializable {
```

Comment:
Let's add a short doc entry for this class and its public methods.

Reply:
added